doc/source/index.rst

```diff
@@ -40,17 +40,202 @@ An example of such a callable class used in trollflow2 is the
+ - `max: <float>` - Maximum allowed Sun-lit coverage in percentages for the product to be generated.
+ - `check_pass: <bool>` - Use orbital parameters to compute the overpass coverage for a polar satellite.
+
+The check can be in three different places, depending on the areas
+being processed:
+
+ - after `create_scene` when area is defined in areas.yaml
+ - after `load_composites` when area is `null`
+   - dynamic area definition is used by `use_min_area: True` or
+     `use_max_area: True`
+   - the original data are to be saved without resampling
+   - `resampler: native` is used
+ - after `resampler` for backwards compatibility, although this wastes time
+
+Add overviews
+*************
+
+Add overview (multi-scale embedded) images in a TIFF file with
+`add_overviews` plugin.  The overviews are configured within the
+`formats` section:
+
+ - `overviews: [ 4, 8 16, 32, 128, 256]`
+
+This plugins should be used after `save_datasets` and before
+`FilePublisher` plugins.
+
+Sun zenith angle check
+**********************
+
+The `sza_check` plugin can be used to check Sun zenith angle (SZA) at
+a given location.  The product will be discarded for the scene if the
+SZA value is out of the configured values.
+
+Options:
+ - `sunzen_check_lon: <float>` - Longitude of the SZA check.
+ - `sunzen_check_lat: <float>` - Latitude of the SZA check.
+ - `sunzen_minimum_angle: <float>` - Minimum required SZA for the product to be processed.  Used for night-time products.
+ - `sunzen_maximum_angle: <float>` - Maximum allowed SZA for the product to be processed.  Used for day-time products.
+
+Area coverage
+*************
+
+The `covers` plugin can be used to check that the received data covers
+the target areas.  If the area coverage is too low (see options
+below), the area is discarded from the processing of this scene.  This
+plugin needs `pytroll-schedule` to be installed.
+
+Options:
+ - `coverage_by_collection_area: False` - If `True`, the
+   `'collection_area_id'` in the incoming message needs to match the
+   name of the target area.
+ - `min_coverage: 0` - Minimum required coverage.  If coverage is less than defined, the data are not processed for this area.  By default process all areas.
+
+Platform name check
+*******************
+
+The `check_platform` plugin can be used to check that the platform
+name in the incoming message is in the configured list of platforms.
+The best place for this plugin is before any data handling is done.
+
+Options:
+ - `processed_platforms: null` - A list of allowed platform names.  By default (`null`) all platforms are processed.
+
+Sensor name check
+*****************
+
+The `check_sensor` plugin can be used to check that the sensor name
+in the incoming message is in the configured list of sensors.  The
+best place for this plugin is before any data handling is done.
+
+Options:
+ - `processed_sensors: null` - A list of allowed sensor names.  By default (`null`) all sensors are processed.
```

Comment:
Just a thought: could these plugins be replaced by a metadata checker ? (Nothing that should be done in this PR obviously)

Reply:
Yes, the same crossed my mind.